ext/opcache/Optimizer/zend_func_info.c

```diff
@@ -240,7 +240,7 @@ static const func_info_t func_infos[] = {
 	F1("system",                       MAY_BE_FALSE | MAY_BE_STRING),
 	F1("escapeshellcmd",               MAY_BE_STRING),
 	F1("escapeshellarg",               MAY_BE_STRING),
-	F1("passthru",                     MAY_BE_NULL | MAY_BE_FALSE),
+	F1("passthru",                     MAY_BE_FALSE),
```

Comment:
Why this change?

Reply:
Oh, that will be a mistake :O I thought that the change in https://github.com/php/php-src/pull/4902/commits/b5f42b0a4706c0e86008be0f271c9f91eda2acd0 will completely eliminate this case, but I forgot about that `passthru()` returns `null` in case of success.

Reply:
(But it's a question for me why I left `null` in the stub then). But in any case, I'll revert the change in opcache.